src/test/java/com/google/api/codegen/testdata/ruby_doc_version_index_library.baseline

```diff
@@ -50,6 +50,13 @@
 # [Product Documentation]: https://cloud.google.com/library
 #
 module Library
+  ##
+  # # Google Example Library API Contents
+  #
+  # | Filename | Description |
+  # | -------- | ----------- |
+  # | library/v1/library_service_client | Accesses the Library::V1::LibraryServiceClient |
```

Comment:
Can the "filename" link to the actual docs page?

Also, can we get a link to the Data Types page ([example](http://googlecloudplatform.github.io/google-cloud-ruby/#/docs/google-cloud/v0.32.0/google/cloud/language/v1/datatypes))?